src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkMultilineFunctionCallInspection.kt

```diff
@@ -0,0 +1,98 @@
+package com.jetbrains.snakecharm.inspections
+
+import com.intellij.codeInspection.*
+import com.intellij.openapi.project.Project
+import com.intellij.psi.*
+import com.intellij.psi.util.elementType
+import com.intellij.refactoring.suggested.startOffset
+import com.jetbrains.python.PyTokenTypes
+import com.jetbrains.python.psi.PyCallExpression
+import com.jetbrains.snakecharm.SnakemakeBundle
+import com.jetbrains.snakecharm.lang.psi.SmkRuleOrCheckpointArgsSection
+
+const val TAB = "    "
```

Comment:
We cannot use such TAB def here, number of spaces in TAB is IDE setting. More over user could use real TAB character instead of spaces.

Reply:
And is only setting for format code features, indents in user file could be different
![image](https://user-images.githubusercontent.com/72933/127317233-a91e8f6a-75b8-4bbd-98a1-601a5b5850fa.png)